fabric-object-builder-api-v1/src/main/java/net/fabricmc/fabric/api/object/builder/v1/entity/FabricEntityTypeBuilder.java

```diff
@@ -55,6 +57,8 @@
 	private EntityDimensions dimensions = EntityDimensions.changing(-1.0f, -1.0f);
 	private ImmutableSet<Block> specificSpawnBlocks = ImmutableSet.of();
 
+	private FeatureSet requiredFeatures = FeatureFlags.VANILLA_FEATURES;
```

Comment:
Shouldn't this be `DEFAULT_ENABLED_FEATURES` instead?

Reply:
Hmm, either works.